integration/test-integration/pom.xml

```diff
@@ -225,7 +225,7 @@
         <dependency>
             <groupId>org.wso2.am.choreo.connect</groupId>
             <artifactId>mock-backend-server</artifactId>
-            <version>1.0.0-rc3-SNAPSHOT</version>
```

Comment:
can you remove this version specification from here and use dependency management/project parent version instead?

Reply:
Using parent version from https://github.com/wso2/product-microgateway/pull/2586/commits/ded1c4aaff63bb7bd5eafa74e5bf2cd320106af8